src/main/java/org/sonarsource/plugins/example/web/MyPluginPageDefinition.java

```diff
@@ -1,33 +1,52 @@
+/*
```

Comment:
Not sure you want the SQ license header in this project ?

Reply:
Indeed 😅 .